docs/pages/access-controls/sso/google-workspace.mdx

```diff
@@ -170,87 +149,225 @@ correct Google Workspace plan.
+Under **Credentials**, select **CREATE CREDENTIALS**, then **OAuth client ID**:
 
-```txt
-https://<address of proxy server>:3080/v1/webapi/oidc/callback
-```
+![Create an OAuth Client ID](../../../img/sso/googleoidc/create-oauth-client-id.png)
+
+Select "Web application" as the Application type, pick a name, then set the
+redirect URI based on the path to your Teleport proxy or cloud tenant:
+<nobr><Var name="https://example.teleport.sh"/>`/v1/webapi/oidc/callback`</nobr>
 
-    ![OAuth client ID creation](../../../img/googleoidc/clientid-creation.png)
+![OAuth client ID creation](../../../img/sso/googleoidc/clientid-creation.png)
 
-  Copy the Client ID and Client Secret from the next screen or by clicking "Download OAuth client".
-    ![OAuth client data](../../../img/googleoidc/clientid-data.png)
+Copy the Client ID and Client Secret from the next screen or click **DOWNLOAD JSON**:
+
+![OAuth client data](../../../img/sso/googleoidc/clientid-data.png)
 
 ### Create a service account
 
-On the [Create a service account](https://console.cloud.google.com/iam-admin/serviceaccounts/create) page, pick a name for your service account. Leave project access grants and user access grants empty.
-    ![service account creation](../../../img/googleoidc/serviceacct-creation.png)
+From the GCP **IAM & Admin** menu select **Service Accounts**. From the kebab
+menu click **CREATE SERVICE ACCOUNT**:
+
+![Create a service account](../../../img/sso/googleoidc/create-service-account.png)
+
+On the [Create a
+service account](https://console.cloud.google.com/iam-admin/serviceaccounts/create)
+page, pick a name for your service account. Leave project access grants and user
+access grants empty.
+
+![service account creation](../../../img/sso/googleoidc/serviceacct-creation.png)
 
-  Click the newly-created account to view its details, and copy the Unique ID for later.
-    ![service account unique ID](../../../img/googleoidc/serviceacct-uniqueid.png)
+Click the newly-created account to view its details, and copy the Unique ID for later.
 
-  Create a new key for the service account, select JSON as the key type, and save the resulting JSON file.
-    ![service account key creation](../../../img/googleoidc/serviceacct-key.png)
-    Later, we will make this JSON available to the Teleport Auth Server via the OIDC Connector configuration, either by referencing a local file or pasting the JSON into the configuration YAML. If you plan to take the first approach, you will need to upload the JSON to the Auth Server.
+![service account unique ID](../../../img/sso/googleoidc/serviceacct-uniqueid.png)
+
+Create a new key for the service account, select JSON as the key type, and save
+the resulting JSON file.
+
+![service account key creation](../../../img/sso/googleoidc/serviceacct-key.png)
+
+Later, we will make this JSON available to the Teleport Auth Server via the
```

Comment:
```suggestion
Later, we will make this JSON available to the Teleport Auth Service via the
```
For consistency with other pages. We're kind of split between "Auth Service" and "Auth Server" in the docs, but the Core Concepts page says "Auth Service".